styles/styles.css

```diff
@@ -0,0 +1,114 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+body {
```

Comment:
- [ ] As pointed out in the screenshot below,  the main requirement of this project is to start working on the mobile version first, but unfortunately, you started with the desktop version of this portfolio rather than the mobile version, Kindly, work on refactoring your code to make it similar to the original design below
![Capture](https://user-images.githubusercontent.com/80895497/148367643-e7b488e8-ef38-45d4-8ef8-a00dec41e30c.PNG)

Kindly, stick to the design as much as possible for better UI (user interface) and this is a good professional practice that every frontend software developer should maintain. Please, make sure to make the hamburger menu closely similar to the [original design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10).

**Current mobile menu**           |  **Original design**
:-------------------------:|:-------------------------:
![](https://user-images.githubusercontent.com/80895497/148366385-6615fb3c-a980-431b-b83f-703209ab35d9.PNG)  |  ![](https://user-images.githubusercontent.com/80895497/148366418-00cb5fbe-d0f9-4e79-95fa-f99f671d723a.PNG)